naucse/routes.py

```diff
@@ -14,13 +14,14 @@
 
 app.jinja_loader = PrefixLoader({
     'templates': FileSystemLoader(os.path.join(app.root_path, 'naucse/templates')),
-    'courses': FileSystemLoader(os.path.join(app.root_path, 'courses'))
+    'courses': FileSystemLoader(os.path.join(app.root_path, 'courses')),
+    'runs': FileSystemLoader(os.path.join(app.root_path, 'runs')),
+    'terms': FileSystemLoader(os.path.join(app.root_path, 'terms')),
+    'videos': FileSystemLoader(os.path.join(app.root_path, 'videos')),
+    'lessons': FileSystemLoader(os.path.join(app.root_path, 'lessons'))
```

Comment:
For the future: please don't add things for which we don't yet have any content in the repo.